doc/man/salt.7

```diff
@@ -403017,10 +403017,6 @@ New in version 3001.
 The file extension to use for a backup of the file if any edit is made.
 Set this to \fBFalse\fP to skip making a backup.
 .TP
-.B dry_run
-If \fBTrue\fP, do not make any edits to the file and simply return the
-changes that \fIwould\fP be made.
-.TP
```

Comment:
We don't need to update the man pages, that is done during the release process.  Would you mind reverting these changes?

Reply:
Thanks for the feedback, it's my first time doing a doc update and I just did a git grep to catch possible changes needed. I've reverted the man page change.